### PR TITLE
Add support for case-insensitive SPF records

### PR DESCRIPTION
**Pull request by jsarenik**
https://tools.ietf.org/html/rfc7208#section-12

> See [RFC5234] for ABNF notation.  Please note that as per this ABNF
> definition, literal text strings (those in quotes) are case-
> insensitive.  Hence, "mx" matches "mx", "MX", "mX", and "Mx".

Fixes GitHub issue jsarenik/spf-tools#124

### CODE REVIEW

File: include/despf.inc.sh [hunk 1]
```diff
@@ -158,6 +158,7 @@ getamx() {
         lookuphost=$(echo $ahost | cut -d\/ -f1)
       fi
     fi
+    mech=$(echo $mech | tr [A-Z] [a-z])
```
Comment (jcbf) [inline include/despf.inc.sh]: Must use comma in tr, like tr '[A-Z]' '[a-z]'.
Without it it, there will be a shell expansion and may have undesired results.
```
echo A | tr [A-Z] [a-z]
a
izeca:tmp jcbf$ touch o
izeca:tmp jcbf$ echo A | tr [A-Z] [a-z]
o
izeca:tmp jcbf$ echo A | tr '[A-Z]' '[a-z]'
a
```

Comment (jsarenik) [inline include/despf.inc.sh]: Good catch José! Thank you!

**Comment by jsarenik** [comment]
@jcbf oops, I didn't use some github magic right and instead of applying your proposed fix I have just force-pushed with my fix... sorry.

**Comment by jsarenik** [comment]
@jcbf thanks for review!

**Pull request merged**